ni_measurementlink_service/_internal/service_manager.py

```diff
@@ -85,24 +87,26 @@ def start(
                 ("grpc.max_send_message_length", -1),
             ],
         )
-        servicer_v1 = MeasurementServiceServicerV1(
-            measurement_info,
-            configuration_parameter_list,
-            output_parameter_list,
-            measure_function,
-        )
-        v1_measurement_service_pb2_grpc.add_MeasurementServiceServicer_to_server(
-            servicer_v1, self.server
-        )
-        servicer_v2 = MeasurementServiceServicerV2(
-            measurement_info,
-            configuration_parameter_list,
-            output_parameter_list,
-            measure_function,
-        )
-        v2_measurement_service_pb2_grpc.add_MeasurementServiceServicer_to_server(
-            servicer_v2, self.server
-        )
+        if _V1_INTERFACE in service_info.provided_interfaces:
```

Comment:
I think this should error if any of the provided interfaces are unrecognized.

What about custom interfaces? 
- This class doesn't support adding custom gRPC servicers to its gRPC server at this time.
- You can create a separate gRPC server for your custom interfaces, but it will have a different listening port than the one we are registering here, so it's incorrect to include it in this method's call to `RegisterService`.

The easiest way to implement this is to iterate over provided_interfaces and do if v1 / elif v2 / else error. 

Also add a test case.